agent/util-scripts/pbench-postprocess-tools

```diff
@@ -93,6 +93,20 @@ fi
 tool_output_dir="$dir/tools-$group"
 mkdir -p $tool_output_dir
 
+export INVENTORY=/tmp/inventory.$$
+pbench-get-hosts $tool_group_dir > ${INVENTORY}
```

Comment:
Could we consider making this a function to just generate an inventory instead of having part of the logic in this script and another part in `pbench-get-hosts`?